components/clarinet-cli/src/frontend/clarinetrc.rs

```diff
@@ -0,0 +1,53 @@
+use std::fs::{self};
+
+use std::env;
+
+#[derive(Serialize, Deserialize, Default)]
+pub struct GlobalSettings {
+    pub enable_hints: Option<bool>,
+    pub enable_telemetry: Option<bool>,
+}
+
+impl GlobalSettings {
+    pub fn get_settings_file_path() -> &'static str {
+        "~/.clarinet/clarinetrc.toml"
+    }
+
+    pub fn from_global_file() -> Self {
+        let home_dir = dirs::home_dir();
+
+        if let Some(path) = home_dir.map(|home_dir| home_dir.join(".clarinet/clarinetrc.toml")) {
```

Comment:
Note for future 2.0 release manager: could be problematic on windows machines.

Reply:
I haven't tested myself but apparently `PathBuf::join` handles the separator correctly on any OS.
I'll leave it as it is